nemo/collections/cv/modules/global_avg_pool2d.py

```diff
@@ -0,0 +1,12 @@
+from nemo.core.classes.module import NeuralModule
```

Comment:
This should go to cv/parts/ rather than cv/modules/

Reply:
done